ser316asu-2018/Schwerin
# Us 27

TESTING INSTRUCTIONS:

- Verify that no errors exist in both new files.
- Verify all implemented functions look sane.
